bees/knowgobee/knowgobeefactory.go

```diff
@@ -0,0 +1,131 @@
+/*
+ *    Copyright (C) 2019 Adaptant Solutions AG
+ *
+ *    This program is free software: you can redistribute it and/or modify
+ *    it under the terms of the GNU Affero General Public License as published
+ *    by the Free Software Foundation, either version 3 of the License, or
+ *    (at your option) any later version.
+ *
+ *    This program is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *    GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *    Authors:
+ *      Paul Mundt <paul.mundt@adaptant.io>
+ */
+
+package knowgobee
+
+import (
+	"github.com/muesli/beehive/bees"
+)
+
+// HelloBeeFactory is a factory for HelloBees.
```

Comment:
Just a nitpick, this should be updated to KnowGoBeeFactory and KnowGoBees 😄

Reply:
Thanks, I thought I caught all of these but I guess not. This is what I get for eyeballing it and doing it manually..